docs/resources/ui/components/built-in-components/color-picker.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: color-picker
+title: Color Picker
+tags: [Actions, Components]
```

Comment:
Add the tag for widget category (like Base Elements or Layout Elements) so they can be grouped together in the generated tag page.

Reply:
Color picker is available as an action only, so added 'Actions' tag. It doesn't fall under any widget category by default.